DataFormats/METReco/interface/BeamHaloSummary.h

```diff
@@ -80,6 +89,9 @@ namespace reco {
     std::vector<int>  EcaliPhiSuspects;
     std::vector<int>  GlobaliPhiSuspects;
 
+    std::vector<std::vector<std::pair<char, CaloTowerDetId> > > ProblematicStrips;
```

Comment:
please keep data members starting with a lower case.
I know, the rest of the file is different, but that's the old stuff that's in conflict with the cms style.